testbed/addons.ts

```diff
@@ -0,0 +1,19 @@
+import cdk = require('@aws-cdk/core');
+import * as eks from '@aws-cdk/aws-eks';
+import { Flux, Repository } from './addons/flux/construct';
+
+export interface AddonsProps extends cdk.StackProps {
+  cluster: eks.Cluster
+  repositories: Repository[];
+}
+
+export class Addons extends cdk.Construct {
+  constructor(scope: cdk.Construct, id: string, props: AddonsProps) {
+    super(scope, id);
+    // Add FluxV2
+    new Flux(this, 'Flux', {
```

Comment:
I think you should be able to pass props directly, since we're using the same fields 
`new Flux(this, 'Flux', props)`

Reply:
I think I want to leave this separate, as we add more addons we might end up having more data to pass arround in Addons interface that is not just specific to Flux ? Thoughts ?

Reply:
Typescript will automatically strip the extra properties.